README.md

```diff
@@ -53,15 +48,15 @@ The following changes can be safely made to the service/database
 
 ## Communicates with
 - Party via a rest call `/party-api/v1/parties` for creating a new business
-- Collection-exercise via rabbit for both `INIT` and `PERSISTED` sample units
+- Collection-exercise via PubSub to inform how the enrichment and distribution of sample units to case is going
 
 ## What it does
 On file upload it creates a new SampleSummary entry in the database with State = INIT
 
 SampleOuboundPublisher - It will then asynchronously parse the sample file and send a message with the routing key `Sample.SampleUploadStarted.binding`
-However this routing key doesnt match the binding key `Sample.SampleDelivery.binding` for the `sample-outbound-exchange` so presumably this message will be descarded (Further analysis required). Collection exercise is listening to the `Sample.SampleDelivery` queue
+However this routing key doesnt match the binding key `Sample.SampleDelivery.binding` for the `sample-outbound-exchange` so presumably this message will be discarded (Further analysis required). Collection exercise is listening to the `Sample.SampleDelivery` queue
 
-CsvIngesterBusiness - Providing the SampleSummary type is "B" it will attempt to parse the uploaded CSV. Foreach line in the sample file create a new callable so this can all be sone asyncronously, pass in a new empty keyset. Our asyncronous callable is now syncronised because opencsv is not actually threadsafe!?!? The empty keyset is then used to store sample units where exceptions are thrown for duplicates.
+CsvIngesterBusiness - Providing the SampleSummary type is "B" it will attempt to parse the uploaded CSV. Foreach line in the sample file create a new callable so this can all be sone asynchronously, pass in a new empty keyset. Our asynchronous callable is now syncronised because opencsv is not actually threadsafe!?!? The empty keyset is then used to store sample units where exceptions are thrown for duplicates.
```

Comment:
This whole section can be deleted actually, the CsvIngesterBusiness is long gone